content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,99 @@
-===========================================================
-Generate leads/opportunities from your website contact page
-===========================================================
+============================
+Generate leads/opportunities
+============================
 
-Automating the lead/opportunity generation will considerably improve
-your efficiency. Any visitor using the contact form on your website will
-create a lead/opportunity in the pipeline.
+Automating lead/opportunity generation can significantly improve your team's efficiency. In this
+article, learn how to use email aliases and website contact forms to automatically generate leads
+in your database.
 
-Use the contact us on your website
-==================================
+Configure email aliases
+=======================
 
-You should first go to your website app.
+Each sales team can use its own email alias to automatically generate leads/opportunities in its
+pipeline. Any emails sent to a sales team's email alias will automatically create a lead or
+opportunity in the pipeline for that particular sales team. You can configure custom email aliases
+on the configuration page for each sales team: :menuselection:`CRM --> Configuration --> Sales
+Teams`.
 
-|image0|\ |image1|
+.. image:: generate_from_website/sales-team-config.png
+   :align: center
+   :alt: Configuring Sales Teams
 
-With the CRM app installed, you benefit from a ready-to-use contact form
-on your Odoo website that will generate leads/opportunities
-automatically.
+Use the Contact Us page on your website
+=======================================
 
-.. image:: media/generate_from_website03.png
+By default, your website's *Contact Us* page displays both your stored company information and
+Odoo's ready-to-use contact form to generate leads/opportunities.
```

Comment:
> Adding a sentence here for clarity about Contact Forms vs. the Contact Us page. Also editing for brevity/clarity.

Removing first sentence since second sentence implies this already (redundant).
Also adding `an` before `opportunity` for grammar.

```suggestion
By default, your website's *Contact Us* page displays Odoo's ready-to-use Contact Form. Whenever
someone submits this form, a lead or an opportunity is generated in your database.
```